__tests__/app.test.js

```diff
@@ -174,6 +174,55 @@ describe('GET /api/users', () => {
     });
 });
 
+describe('GET /api/articles', () => {
+    test('should return an array', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            expect(Array.isArray(body)).toBe(true);
+            expect(body).toHaveLength(12)
+        });
+    });
+    test('should return status 200 and user array with correct properties', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            body.forEach((article) => {
```

Comment:
All of this can be combined into one forEach with no return statement. If you are going to use a forEach in this case, it is best practice to assert that the array has at least one item in it.